packages/extension-ui/src/partials/HeaderWithSteps.tsx

```diff
@@ -85,3 +82,7 @@ function HeaderWithSteps({
 }
 
 export default React.memo(HeaderWithSteps);
+
+const MarginLessHeader = styled(Header)`
+  margin-bottom: 0;
+`;
```

Comment:
We could consider separate export from Header, sth like:
```
export const MarginLessHeader;
export default Header;
```
But not important at all, so... :shrug:

Reply:
Yeah, now that the pattern emerged after the refactor it does seem so... But even better would be to get rid of the margin from the header and have it added in the children (so kind of reverse the logic). Even yet better would be to integrated the header with the content ans scroll wrapper so that the views don't have to care about neither the header nor the margins. I just don't know know at which level to stop the refactors xP

Reply:
Ok, this header margin is [whole a lot problematic](https://github.com/Cardinal-Cryptography/aleph-zero-signer/pull/105#discussion_r1253468486). I'm replacing it with something else.